### ISSUE TITLE
 [BUG] latest published versions contain bogus LICENSE files

### ISSUE THREAD

**Issue by decathorpe** [plotters-rs/plotters#649]
**Describe the bug**

This is a followup for https://github.com/plotters-rs/plotters/issues/392 and https://github.com/plotters-rs/plotters/pull/477.

It looks like you `cargo publish` was run for v0.3.7 on a system that does not support symbolic links - the LICENSE files contain just the string `../LICENSE` and not the actual license text.

If you are going to be publishing from Windows again, symbolic links will not be enough to include the license files, but actual copies of the files will be necessary to have cargo pick them up correctly.

**To Reproduce**

Download plotters-* crates from crates.io and inspect them. The LICENSE files contain just the string `../LICENSE` instead of the license text.

**Version Information**

- plotters v0.3.7
- plotters-backend v0.3.7
- plotters-bitmap v0.3.7
- plotters-svg v0.3.7

**Comment by decathorpe** [comment]
No, cargo definitely does support this ... I tested running `cargo package` on Fedora 41, and the resulting .crate file contains the correct LICENSE file, and not just a file that contains the string  `../LICENSE`.

Looks like something went wrong when the crates for v0.3.7 were published - either the local checkout of the git repo was corrupted, or you used a buggy version of cargo.

**Comment by AaronErhardt** [comment]
Mhh, strange. Maybe you have to use `cargo publish -p ...` instead of `cd ... && cargo publish` because cargo does not load symlinks from outside of the current pwd?

**Comment by decathorpe** [comment]
I don't think so. I just tried both, and both worked :(

**Comment by AaronErhardt** [comment]
Then I'm out of ideas why it didn't work. I should have a regular Fedora setup with a recent Rust version via rustup and without notable configuration changes. I'll have a look at this tomorrow.

**Comment by AaronErhardt** [comment]
I just tried both `cargo package` variants and both didn't work for me. Not sure why honestly. As I said earlier, I just have a regular Fedora setup...

**Comment by decathorpe** [comment]
Did you try from a fresh git clone of this repo? Maybe your local checkout is corrupted somehow.

**Comment by AaronErhardt** [comment]
Thanks for the good idea!
That actually worked. I guess the next release will fix the issue then automatically.

**Comment by decathorpe** [comment]
Good to know. Thanks for checking!
